shell/platform/windows/flutter_project_bundle.cc

```diff
@@ -80,8 +80,16 @@ UniqueAotDataPtr FlutterProjectBundle::LoadAotData(
 
 FlutterProjectBundle::~FlutterProjectBundle() {}
 
+void FlutterProjectBundle::SetSwitches(std::vector<std::string> switches) {
```

Comment:
This should pass by `const &` and then copy, rather than copy here and move internally, as that's a much more common pattern (and this isn't going to be a huge amount of data where allowing the micro-optimization of the caller `std::move`ing is necessary).